test_autoarray/mask/test_mask_2d_util.py

```diff
@@ -1843,6 +1843,37 @@ def test__different_shape_and_masks(self):
         ).all()
 
 
+class TestUnmaskedPixels:

```

Comment:
Whats the point in this class given that it only contains a single test?

Reply:
Old James did class for all tests, I'm slowly refactoring all the code to single tests as I've gone off using classes for most things. So this will get sorted eventually.